CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 -	[#21992](https://github.com/influxdata/influxdb/pull/21992): fix: systemd-start script should be executable by group and others
 -	[#22019](https://github.com/influxdata/influxdb/pull/22019): fix: error instead of panic when enterprise tries to restore with OSS
 -	[#22040](https://github.com/influxdata/influxdb/pull/22040): fix: copy names from mmapped memory before closing iterator
+-	[#22104](https://github.com/influxdata/influxdb/issues/22104): fix: ensure log formatting (JSON) is respected
```

Comment:
I think this needs updated to link to the pull request instead of the issue.

Reply:
Whoops!

Reply:
Friday afternoon and too many browser tabs open...  Fixed.